src/TransitionSystems/location_id.rs

```diff
@@ -61,6 +61,29 @@ impl LocationID {
             None
         }
     }
+
+    pub(crate) fn get_unique_string(&self) -> String {
```

Comment:
Hmm ved ikke om der skal docs her?

Reply:
I have made the function accessible only in the super module (`TransitionSystem`), I think that's fine, wby?